values/src/test/java/org/spine3/time/LocalDatesShould.java

```diff
@@ -51,53 +51,46 @@ public void obtain_date_from_year_month_day_values() {
         final int year = 1976;
         final int month = 3;
         final int day = 1;
-        final LocalDate birthday = LocalDates.of(1976, MonthOfYear.APRIL, 1);
         final Calendar calendar = Calendar.getInstance();
         calendar.set(year, month, day);
+        final LocalDate birthday = LocalDates.of(1976, MonthOfYears.getMonth(calendar), 1);
+
         assertEquals(calendar.get(Calendar.YEAR), birthday.getYear());
         assertEquals(calendar.get(Calendar.MONTH) + 1, birthday.getMonthValue());
         assertEquals(calendar.get(Calendar.DAY_OF_MONTH), birthday.getDay());
     }
 
     @Test
-    public void obtain_date_in_past_after_specified_number_of_years() {
-        final int year = 2007;
-        final int month = 1;
-        final int day = 28;
-        final LocalDate today = LocalDates.of(2008, MonthOfYear.FEBRUARY, 29);
-        final LocalDate yearAgo = LocalDates.minusYears(today, 1);
+    public void obtain_date_in_past_before_specified_number_of_years() {
         final Calendar calendar = Calendar.getInstance();
-        calendar.set(year, month, day);
+        calendar.add(Calendar.YEAR, -1);
+        final LocalDate today = LocalDates.now();
+        final LocalDate yearAgo = LocalDates.minusYears(today, 1);
+
 
         assertEquals(calendar.get(Calendar.YEAR), yearAgo.getYear());
```

Comment:
Please extract static method `getYear()` and the like so that assertion arguments are shorter.